sageresearch-mpower/src/main/java/org/sagebionetworks/research/mpower/sageresearch/inject/MPowerSageResearchModule.java

```diff
@@ -57,4 +63,10 @@
 public abstract class MPowerSageResearchModule {
     @ContributesAndroidInjector
     abstract PerformTaskActivity contributePerformTaskActivityInjector();
+
+    @Provides
+    @ResearchDatabaseFilename
+    static String provideResearchDBFilename() {
+        return "SageResearch-mPower";
```

Comment:
Seems a little but obfuscated to provide a db name this way.  Why not just a String constant?